From 2ee1e197f73b8ab6819784eb1abdf7a7e2a31b45 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Fri, 2 Dec 2016 23:26:06 +0000
Subject: [PATCH] OS-5856 Want support for XXV710 Reviewed by: Jerry Jelinek
 <jerry.jelinek@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 overlay/generic/etc/driver_aliases | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/overlay/generic/etc/driver_aliases b/overlay/generic/etc/driver_aliases
index 35dfbf9d..2bc1c105 100644
--- a/overlay/generic/etc/driver_aliases
+++ b/overlay/generic/etc/driver_aliases
@@ -497,6 +497,8 @@ i40e "pciex8086,1584"
 i40e "pciex8086,1585"
 i40e "pciex8086,1586"
 i40e "pciex8086,1589"
+i40e "pciex8086,158a"
+i40e "pciex8086,158b"
 i915 "pci8086,2562"
 i915 "pci8086,2572"
 i915 "pci8086,2582"
-- 
2.21.0

